code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2546,7 +2546,7 @@
 	name = "Medical Nanobots"
 	id = MEDNANOBOTS
 	description = "Microscopic robots intended for use in humans. Configured for rapid healing upon infiltration into the body."
-	reagent_state = LIQUID
+	reagent_state = SOLID
```

Comment:
From what I could see in the code. 2 is defined as a LIQUID and in the analyze bit of the code the Odysseus checks if the reagent_state is 2.
This is also the reason as to why chloral(Despite obviously being a liquid) wasn't analyzeable, since it is also a SOLID in the code.
The Odysseus had no blacklist or whitelist of any kind and it seems that reagent_state is merely fluff other than Odysseus scanning. So this was a convenient fix.